collect_app/src/main/res/layout/geoshape_layout.xml

```diff
@@ -42,6 +42,14 @@
         android:layout_height="wrap_content"
         android:paddingTop="6dip">
 
+        <android.support.v7.widget.AppCompatImageButton
+            android:id="@+id/backspace"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:padding="15dp"
+            android:layout_margin="5dp"
+            android:src="@drawable/ic_backspace_arrow_selector" />
```

Comment:
Why `android:src` not `app:srcCompat` like in other buttons. it's a vector file so we should rather use `app:srcCompat`.

Reply:
Fixed, thanks!